HdcpProfile/HdcpProfile.cpp

```diff
@@ -107,24 +109,24 @@ namespace WPEFramework
             Utils::IARM::init();
 
             IARM_Result_t res;
-            IARM_CHECK( IARM_Bus_RegisterEventHandler(IARM_BUS_DSMGR_NAME,IARM_BUS_DSMGR_EVENT_HDMI_HOTPLUG, dsHdmiEventHandler) );
-            IARM_CHECK( IARM_Bus_RegisterEventHandler(IARM_BUS_DSMGR_NAME,IARM_BUS_DSMGR_EVENT_HDCP_STATUS, dsHdmiEventHandler) );
+            IARM_CHECK( Utils::Synchro::RegisterLockedIarmEventHandler<HdcpProfile>(IARM_BUS_DSMGR_NAME,IARM_BUS_DSMGR_EVENT_HDMI_HOTPLUG, dsHdmiEventHandler) );
+            IARM_CHECK( Utils::Synchro::RegisterLockedIarmEventHandler<HdcpProfile>(IARM_BUS_DSMGR_NAME,IARM_BUS_DSMGR_EVENT_HDCP_STATUS, dsHdmiEventHandler) );
```

Comment:
Why the changes done for only two plugins for migrating to Utils::Synchro? Are you facing any isisue with only above two plugin if not do we have ticket to migrate other plugins can we merge all together?

Reply:
As we discussed in the call yesterday, we were using 4 plugins in our base stack.  HDMICEC, Display settings changes are already merged as a part of another pull request (https://github.com/rdkcentral/rdkservices/pull/4232).  This PR covers HdcpProfile and Xcast.  We do not have any environment to edit, compile and test other plugins.
So, please review and merge these changes and probably as agreed you can have another internal tracking ticket for implementing these type of changes on other plugins.